config/environments/test.rb

```diff
@@ -26,47 +26,56 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 
-OpenProject::Application.configure do
+require 'active_support/core_ext/integer/time'
+
+# The test environment is used exclusively to run your application's
+# test suite. You never need to work with it otherwise. Remember that
+# your test database is "scratch space" for the test suite and is wiped
+# and recreated between test runs. Don't rely on the data there!
+
+Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
   # Access to rack session
   config.middleware.use RackSessionAccess::Middleware
 
-  # The test environment is used exclusively to run your application's
-  # test suite. You never need to work with it otherwise. Remember that
-  # your test database is "scratch space" for the test suite and is wiped
-  # and recreated between test runs. Don't rely on the data there!
-  #
   # Spring requires to have the classes reloaded. On the CI or when Spring is
   # disabled, it does not need to happen.
-  config.cache_classes = ENV['CI'].present? || ENV['DISABLE_SPRING'].present?
+  config.enable_reloading = !(ENV['CI'].present? || ENV['DISABLE_SPRING'].present?)
```

Comment:
Will this be simpler to read?
```suggestion
  config.enable_reloading = %w[CI DISABLE_SPRING].none?{ |name| ENV[name].present? }
```